### PR TITLE
LPS-118805 - Remove internal loading state for iframe

### PR DESCRIPTION
**Pull request by bryceosterhaus**
So I was able to narrow down the percieved performance issue to being due to the `hide` class on the iframe waiting for the onLoad to fire. The onLoad only fires after all resources has been loaded, whereas the previous AUI implementaion didnt hide the iframe. However, by removing the `.hide` class we get a flicker of the styles. The flicker comes from the `.dialog-iframe-popup` class being applied `onLoad`. This wouldn't be an issue, but the url being loaded in the modal doesn't include a `p_p_id` to properly namespace the `bodyCssClass`.

So long story short, this fixes the percieved performance of the modal loading, but I'm not quite sure how we address the iframe urls to properly have the `p_p_id` param for the body class to properly get added.

### CODE REVIEW

**Comment by bryceosterhaus** [comment]
ci:test:relevant
